source-code/plugins/json/src/plugin.ts

```diff
@@ -336,6 +336,7 @@ async function saveMessages(args: {
 		}
 		for (const [languageTag, value] of Object.entries(storage)) {
 			const pathWithLanguage = args.settings.pathPattern.replace("{languageTag}", languageTag)
+			await args.nodeishFs.mkdir(pathWithLanguage.split("/").slice(0, -1).join("/"), { recursive: true })
```

Comment:
I'm sure I already sayed it a few times:
**Plugins must work also if a file and folder does not exist yet!**
I fixed it quickly to be able to continue working on the SDK. @inlang/plugins please add a test and fix it in the other plugins too.

Reply:
I thought we do not want to have empty resource files any more. 
In the editor we look in the config (or add tag to it) to show editor then wenn u edited something u can push it with content. How is it for the SDK?

Reply:
> I thought we do not want to have empty resource files any more.

This has nothing to do with empty resource files..

In the SDK, if there are no messages yet, I will create some examples. And therefore `saveMessages` should not fail.